<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QCheckBox" name="checkBoxCopyFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>277</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранять файлы , в случае ошибки шифрования</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddIO">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/png/res/add.jpg</normaloff>:/png/res/add.jpg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить флешку для ключа</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSerialUSB">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Серийный номер</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/png/res/ok.png</normaloff>:/png/res/ok.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/png/res/cancel.png</normaloff>:/png/res/cancel.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSerialHDD">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Серийный номер</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddIO_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/png/res/add.jpg</normaloff>:/png/res/add.jpg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить жесткий диск для ключа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Алгоритм шифрования</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAlg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AES 128bit CBC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AES 192bit CBC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AES 256bit CBC</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
